web/client/actions/router.js

```diff
@@ -17,3 +20,8 @@ export function goToPage(page, router) {
         page
     };
 }
+
+export function goToHomePage() {
+    const homePath = ConfigUtils.getMiscSetting('homePath', '/');
```

Comment:
can we use the goToPage action instead of push?

Reply:
In that case, we may need to pass router as a param from every place we trigger it